reference/6/Microsoft.PowerShell.Management/Remove-Item.md

```diff
@@ -86,6 +86,42 @@ The path is specified, but the optional parameter name (*Path*) is omitted.
 The *Recurse* parameter deletes all of the contents of the OldApp key recursively.
 If the key contains subkeys and you omit the *Recurse* parameter, you are prompted to confirm that you want to delete the contents of the key.
 
+### Example 6 - Deleting files with special characters
```

Comment:
Please consider changing this to a non-Windows example.

Reply:
Later. I don't have a test environment for that.